src/Sarif.Viewer.VisualStudio/Models/FeedbackType.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sarif.Viewer.Models
+{
+    public enum FeedbackType
+    {
+        None = 0,
+        UsefulResult = 1,
```

Comment:
> = 1 [](start = 20, length = 4)

you don't need to assign 1 -> 6. Remove, plase

Reply:
done